eth_utils/logging.py

```diff
@@ -77,7 +85,10 @@ class HasLogger(_BaseHasLogger):
     pass
 
 
-class _BaseHasExtendedDebugLogger(HasLogger[ExtendedDebugLogger]):  # type: ignore
+HasExtendedDebugLoggerMeta = HasLogger.replace_logger_class(ExtendedDebugLogger)
```

Comment:
Does it make more semantic sense to use `HasLoggerMeta` here? (assuming both options work)